nginx/conf.d/default.conf.template

```diff
@@ -6,21 +6,13 @@ server {
         root   /usr/share/nginx/html;
         index  index.html index.htm;
     }
+    
+    location /api {
+        proxy_pass  http://127.0.0.1:8080;
```

Comment:
Changing this to `http://localhost:8080` might be better for clarity and consistency if we are going to use `localhost` in general.